tests/test_flow_studio_ui_ids.py

```
@@ -109,7 +113,13 @@ def extract_uiids_from_html(html: str) -> List[Tuple[str, int]]:
                 continue
             uiids.append((value, line_num))
 
-    return uiids
+    # Deduplicate extracted uiids while preserving the first encountered line number
+    deduped_uiids = {}
+    for value, line_num in uiids:
+        if value not in deduped_uiids:
+            deduped_uiids[value] = line_num
+
+    return [(value, line_num) for value, line_num in deduped_uiids.items()]
```

Ruff (0.15.5)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)